### PR TITLE
Moved the batch builder factory configuration into the sawtooth.json configuration files

### PR DESCRIPTION
**Pull request by mtrubs**
https://github.com/hyperledger/caliper/issues/189

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,15 +60,11 @@
...
       "src/comm/client/**",
-      "src/comm/rate-control/pidRate.js",
-      "src/comm/rate-control/rateControl.js",
-      "src/comm/rate-control/rateInterface.js",
       "src/composer/**",
       "src/contract/**",
       "src/fabric/**",
       "src/gui/**",
       "src/iroha/**",
-      "src/sawtooth/**",
```
Comment (mtrubs) [inline package.json]: I could not tell if there was a reason for all of these excludes or not. I left most. The rate-control files above actually already have some coverage.

File: package.json [hunk 2]
```diff
@@ -19,6 +19,10 @@
   },
   "engine-strict": true,
   "engineStrict": true,
+  "files": [
+    "*",
+    "src/comm/template/report.html"
+  ],
```
Comment (mtrubs) [inline package.json]: I needed to add this so that the template html file gets included on an `npm install`

**Comment by hurf** [comment]
Thanks @mtrubs for the contribution. Can you squash the commits? And @feihujiang please take a look.

**Comment by mtrubs** [comment]
@hurf squashed.

**Comment by feihujiang** [comment]
LGTM, Thanks.

**Pull request merged**